### ISSUE TITLE
[$10][MSFT-253] "Sorry!! Your SSO provider is not yet supported." error message: foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2905]
**Steps to Reproduce**
1. Go to: https://accounts.topcoder.com/sso-login/?app=member
2. Type "test" in 'email or handle' text-box
3. Click on "Continue" button
4. Right click > Inspect Elements 
5. Check the "Sorry!! Your SSO provider is not yet supported." error message

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change text color to #EB044E. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: #FF0066
Background Color: #FFFFFF
Contrast Ratio: 3.85:1
=> Therefor, "error message" foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F32

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096230 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096230 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
Please assign me

**Comment by r0hit-gupta** [comment]
PR at https://github.com/appirio-tech/accounts-app/pull/235

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096230 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Dara-K** [comment]
@r0hit-gupta Kindly use #df1e07 for the red error

**Comment by r0hit-gupta** [comment]
@Dara-K done. PR at https://github.com/appirio-tech/accounts-app/pull/240

**Comment by tosha5252** [comment]
Verified

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096725 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096725 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by lakshmiathreya** [comment]
Verified color set to #df1e07 for the Red Error message - Win10/Chrome and FF; MacOS/Chrome and Edge.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096725<br/><br/>```This is an automated message for crazyk via Topcoder X```
